src/tools/sssctl/sssctl_config.c

```diff
@@ -34,6 +34,37 @@
 
 
 #ifdef HAVE_LIBINI_CONFIG_V1_3
+
+static char *sssctl_config_snippet_path(TALLOC_CTX *ctx, const char *path)
+{
+    char *tmp;
```

Comment:
In sssctl_config_check() all variables are initialized by NULL, I think we should stick to this convention here.